### DISCUSSION TITLE
Having an `updatedOn` field

### DISCUSSION THREAD

**Discussion by Ariber-Yadier** [decaporg/decap-cms#7017] [Q&A]
Can someone with a better understanding of Decap CMS tell me how would I add an `updatedOn` field that automatically updates?

I tried using the following code, but sometimes it works and sometimes it's just empty:
```javascript
CMS.registerEventListener({
  name: 'preSave',
  handler: ({ entry }) => {
    const updatedOn = entry.get('updatedOn');
    return entry.get('data').set('updatedOn', updatedOn);
  }
})
```

**Comment by drzax** [comment]
It looks like this code is simply setting `updatedOn` to its existing value. If you want it to be the current date/time, something like this is probably what you want:

```js
CMS.registerEventListener({
  name: 'preSave',
  handler: ({ entry }) => {
    return entry.get('data').set('updatedOn', new Date().toISOString());
  }
})
```
